packages/commonwealth/client/scripts/views/components/AddToHomeScreenPrompt/AddToHomeScreenPrompt.tsx

```diff
@@ -0,0 +1,106 @@
+import React, { useEffect, useState } from 'react';
+import { CWShareIcon } from '../component_kit/cw_icons/cw_custom_icons';
+import './AddToHomeScreenPrompt.scss';
+
+interface AddToHomeScreenPromptProps {
+  isIOS: boolean;
+  isAndroid: boolean;
+}
+
+const AddToHomeScreenPrompt: React.FC<AddToHomeScreenPromptProps> = ({
```

Comment:
let's get rid of React.FC - we dont use it

`const AddToHomeScreenPrompt = ({isIOS, isAndroid}:AddToHomeScreenPromptProps) => {}`